client/store/modules/repository/comments/mutations.js

```diff
@@ -10,11 +10,19 @@ const sseUpdate = (state, comment) => {
   Vue.set(state.items, existing._cid, { ...existing, ...data });
 };
 
+const setSeenComment = (state, { recentComment, activityUid }) => {
```

Comment:
Consider renaming to markSeenComments (since there might be multiple marked as seen). Consider changing payload to `{ activityUid, lastCommentAt }`.  Document above how this thing is working.